src/main/java/com/axterior/jobs/JobsPage.java

```diff
@@ -246,7 +246,35 @@ public void searchJobByDepartment(String department) {
         chooseFromList(listOfAllFilters, department);
         applyFiltersButton.click();
         pause(2);
-        System.out.println(totalNumberOfJobsFound.getText());
+        System.out.println("Search Results under searchJobByDepartment Test, For " + department + " department: " + totalNumberOfJobsFound.getText() + "\n");
+        pause(5);
+    }
+
+
+    public void searchJobByProjectName(String projectName) {
+        expandSideBarIcon.click();
+        jobsModule.click();
+        pause(2);
+        resetFiltersButton.click();
+        waitForVisibilityOfAllElements(listOfAllFilters);
+        chooseFromList(listOfAllFilters, projectName);
+        applyFiltersButton.click();
+        pause(2);
+        System.out.println("Search Results under searchJobByProjectName Test, For " + projectName + " project: " + totalNumberOfJobsFound.getText() + "\n");
+        pause(5);
+    }
+
+
```

Comment:
The two methods searchJobByStatus and searchJobByDepartment have similar functionality and structure. To reduce code duplication and improve maintainability, we can combine the common logic into a single method.

Reply:
I wanted to keep the filters independent of each other that is why I am compromising the code duplication in return of better filter and method identification. 
If you still like me to merge them into one function then I can do it. Let me know!